service/src/main/resources/application.yml

```diff
@@ -15,6 +16,13 @@ externalcreds:
     key-id: ${KMS_KEY_ID:ecm-dev-kms-symmetric-key-for-ssh-key-encrypt-decrypt}
     key-ring-location: ${kMS_KEY_LOCATION:us-central1}
     ssh-key-pair-refresh-duration: ${KMS_KEY_REFRESH_DURATION:90d}
+  nih-credentials-sync-config:
+    check-frequency: ${NIH_ALLOWLIST_CHECK_CRON_STRING:-} # For once-per-hour, use @hourly
```

Comment:
`-` disables the scheduled job, so that we don't run it until the deployment shenanigans are all done.